{"name": "Courier re-registration", "status": "passed", "description": "Courier registration with existing data, expected error", "start": 1703109053731, "stop": 1703109054109, "uuid": "196d00c4-35de-4609-9468-1126c1d67f55", "historyId": "29cc65194e33c07947ad0ee663bf9e6b", "testCaseId": "29cc65194e33c07947ad0ee663bf9e6b", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_duplication_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-N69F3B6"}, {"name": "thread", "value": "8892-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}